crates/rerun-cli/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+name = "rerun-cli"
+authors.workspace = true
+categories = ["visualization", "computer-vision"]
+default-run = "rerun"
+description = "Log images, point clouds, etc, and visualize them effortlessly"
+edition.workspace = true
+homepage.workspace = true
+include.workspace = true
+keywords = ["mesh", "plotting", "point-cloud", "robotics", "visualization"]
+license.workspace = true
+publish = true
+readme = "README.md"
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+[package.metadata.docs.rs]
+all-features = true
+targets = ["x86_64-unknown-linux-gnu", "wasm32-unknown-unknown"]
+
+
+[features]
+# The default is what the user gets when they call `cargo install rerun-cli`,
+# so wer have all the bells and wistles here
+default = ["native_viewer", "web_viewer"]
+
+## Support spawning a native viewer.
+## This adds a lot of extra dependencies, so only enable this feature if you need it!
+native_viewer = ["rerun/native_viewer"]
+
+## Support serving a web viewer over HTTP.
```

Comment:
is there a deeper meaning to ## or is it a typo?

Reply:
There is a deeper meaning if one uses https://docs.rs/document-features/latest/document_features/ - which I should :)